tox.ini

```diff
@@ -0,0 +1,20 @@
+[tox]
+envlist =
+    py27
+    py37
+    py38
+    py39
+
+[testenv]
+usedevelop = true
+extras = test
+deps =
+    py27: pytest >= 4.6, <5
+    !py27: pytest
+    pytest-cov
+    pytest-flake8
+    gocept.pytestlayer
+    py27: configparser < 5
+    py27: zipp < 2
+commands =
+    pytest []
```

Comment:
I would like to have the `{posargs}` parameter here to be able to select individual test cases later on.

Reply:
This is what `[]` does but it is a shorthand … I never can keep the long version in mind.

Reply:
And I forgot the short one :thinking: